addOns/openapi/src/test/resources/org/zaproxy/zap/extension/openapi/PetStore_defn.json

```diff
@@ -0,0 +1,1040 @@
+{
```

Comment:
It would be better if the test definitions were reduced to the minimum required for the test (otherwise we end up having several tests testing the same thing).

Reply:
@thc202 I've reduced test definitions to what I can determine is appropriate for new and existing cases. Please let me know if there are other optimizations you had in mind that I may have missed. Thanks.

Reply:
This file can be removed and used the existing one with the same contents.

Reply:
There are other models/endpoints that can be removed as well.